src/bundle/Resources/views/content/content_edit/content_edit_blocks.html.twig

```diff
@@ -0,0 +1,54 @@
+{% extends 'EzPlatformAdminUiBundle:content/content_edit/base:content.html.twig' %}
+
+{% trans_default_domain 'content_edit' %}
+
+{% block details %}
+    {% if baseLanguage is defined and baseLanguage is not null %}
+        <h2 class="text-muted">{{ 'editing_in_language_based_on'|trans({'%contentName%': content.name, '%languageName%': language.name, '%baseLanguageName%': baseLanguage.name})|desc('Editing - %contentName% in %languageName% based on %baseLanguageName%') }}</h2>
+    {% else %}
+        <h2 class="text-muted">{{ 'editing_in_language'|trans({'%contentName%': content.name, '%languageName%': language.name})|desc('Editing - %contentName% in %languageName%') }}</h2>
+    {% endif %}
+    <h1>
+        <svg class="ez-icon ez-icon-{{ contentType.identifier }}">
+            <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#{{ contentType.identifier }}"></use>
+        </svg>
+        {{ content.name }}
+    </h1>
+
+    <div class="small">
+        {{ contentType.name }} / {{ 'created_by'|trans({'%name%': 'Administrator'})|desc('Created by %name%') }} / {{ content.versionInfo.creationDate|localizeddate('medium', 'medium', app.request.locale) }} / {{ 'content_id'|trans({'%contentId%': content.id})|desc('Content ID: %contentId%') }}{% if content.versionInfo.contentInfo.mainLocationId %}, {{ 'location_id'|trans({'%locationId%': content.versionInfo.contentInfo.mainLocationId})|desc('Location ID: %locationId%') }}{% endif %}
```

Comment:
Hardcoding Administrator seems wrong to me, but maybe I'm missing something.

Reply:
Yes, the name should come from User/Owner object.